_posts/newsletters/2023-01-24-newsletter.md

```diff
@@ -0,0 +1,215 @@
+* [US-RSE Outreach](#outreach)
+* [US-RSE Financials](#financials)
+* [Get Involved](#involved)
+* [Recent Job Postings](#jobs)
+
+    
+    
+    
+<a name="elections"></a>
+# **Steering Committee Elections**
+    
+Thank you to the nine excellent [candidates](https://us-rse.org/2022-11-18-sc-candidates/) who shared their vision for the US-RSE Association's future and the 135 US-RSE members who voted in the 2022 Steering Committee election.  The four candidates elected to a two-year term starting January 2023 are:
+
+* Julia Damerow
+* Rinku Gupta
+* Kenton McHenry
+* Miranda Mundt
+
+We're excited to welcome three new members to the [Steering Committee](https://us-rse.org/about/steering-committee/) and to have Julia back for two more years. 
+
+Thanks to our other excellent candidates Keith Beattie, Aaron Holmes, Lauren Milechin, Lance Parsons, and Joshua Teves for making the election a success. All of them have helped to grow US-RSE and make it successful. We look forward to their continued contributions.
+
+We extend a special thank you to Lance Parsons who was a founding member of the Steering Committee and has volunteered hundreds of hours over the past four years creating US-RSE. We especially want to acknowledge his continuing work with the Diversity, Equity, and Inclusion working group and the Code of Conduct review effort.
+
+We also want to thank two outgoing Steering Committee members who did not participate in the election, Daniel Katz and Chris Hill. Both are founding members of the US-RSE Association and US-RSE Steering Committee. They brought decades of experience to establish US-RSE, connect it to the broader research software community, and set US-RSE up for a bright future. Both Dan and Chris are still leading various community activities and initiatives, so we can expect to continue to benefit from their experience and leadership. 
+   
+<a name="conference2023"></a>
+# **First US-RSE Conference in 2023 in Chicago**
+We are excited to organize the first US-RSE Conference, October 16-18, 2023 at the University of Illinois, Chicago!
+
+Thank you to all those who volunteered to act as a chair for the organizing committee. The chairs are:
+
+- **General**: Sandra Gesing, Julia Damerow
+- **Diversity**: Ludovico Bianchi, Tab Memmott, Lance Parsons
+- **Outreach**: Caleb Jackson, Miranda Mundt
+- **Program**: Kenton McHenry
+- **Publication**: Charles Ferenbaugh, Torin White
+- **Sponsor**: Angela Herring, Mahmood Shad, Chris Hill
+- **Student**: Nicole Brewer, Carrie Brown
+
+If you'd like to volunteer to a particular committee, please let us know!
+
+<!-- Can we make a form for this? Or reuse the previous one with different options? -->
+
+
+<a name="sc-update"></a>
+# **Steering Committee Recap**
+
+In November and December, the Steering Committee has been discussing moving to a new fiscal sponsor that will be able to better support our future financial sustainability.  The Steering Committee voted to apply to a new sponsor.
```

Comment:
```suggestion
In November and December, the Steering Committee discussed moving to a new fiscal sponsor that would be able to better support our future financial sustainability.  The Steering Committee voted to apply to a new sponsor.
```

Reply:
While I agree, we've only applied, so we are under review. We can announce if/when we are approved.